### ISSUE TITLE
Error in Nonlinear Time Continuous Problem Tutorial

### ISSUE THREAD

**Issue by iliailmer** [SciML/DataDrivenDiffEq.jl#375]
When running code from [this](https://datadriven.sciml.ai/stable/examples/4_nonlinear_continuous_system/#nonlinear_continuos) tutorial, I run into an error at the step defining `ContinuousDataDrivenProblem`:

```julia
> prob = ContinuousDataDrivenProblem(solution)
```

The error is:
```julia
ERROR:"No matching function wrapper was found!"
....
```

I followed the tutorial step-by-step, here are the package versions I have:
```julia
  [2445eb08] DataDrivenDiffEq v0.8.5
  [0c46a032] DifferentialEquations v7.4.0
  [f213a82b] HomotopyContinuation v2.6.4
  [961ee093] ModelingToolkit v8.23.0
  [1dea7af3] OrdinaryDiffEq v6.27.2
  [91a5bcdd] Plots v1.34.1
  [f27b6e38] Polynomials v3.2.0
  [cf7bdac0] SIAN v1.3.1
  [220ca800] StructuralIdentifiability v0.4.1
```
What could be the issue here?

**Comment by ChrisRackauckas** [comment]
It would probably help to do: `problem = ODEProblem{true, SciMLBase.FullSpecialize}(slow_manifold, u0, tspan, p)`. That should be done internally to DataDrivenDiffEq.jl

See https://sciml.ai/news/2022/09/21/compile_time/ for details.

**Comment by iliailmer** [comment]
Thank you, it helped! I still get a Koopman error, I think it's similar to #370 .

**Comment by bgroenks96** [comment]
I also got this (resolved by specifying `FullSpecialize`) and the same error from #370.

**Comment by ChrisRackauckas** [comment]
Yeah, we need to make it so that DataDrivenDiffEq does it automatically internal to the package, I just haven't gotten around to it.

**Comment by AlCap23** [comment]
On that note: Creating a problem from a DESolution without using the interpolation sometimes fails given that I attempt to call the fully specialised function.
